msal-dotnet-articles/advanced/managed-identity.md

```diff
@@ -17,6 +17,8 @@ A common challenge for developers is the management of secrets, credentials, cer
 * [Azure Cloud Shell](/azure/cloud-shell/overview)
 * [Azure Service Fabric](/azure/service-fabric/service-fabric-overview)
 
+> Service Fabric requires a certificate validatidation which is supported [4.62.0](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/CHANGELOG.md#4620) onwards. The validation is available for Net 4.7.1 and greater, Net Standard and Net Core.
```

Comment:
why does service fabric needs this validation?

Reply:
This is the protocol to add the cert validation to the http client using the thumbprint available in the env variable. For more info [see](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4462)